news/2019-09-27-mappers-guild-september-newspost.md

```diff
@@ -0,0 +1,131 @@
+
+The Mappers' Guild is a group of experienced mappers which its goal is to produce official content for the game, exclusively from Featured Artist songs while rewarding mappers with points depending on their individual contribution. The quests are usually artist-specific, but can sometimes be thematic, such as Vocaloid-themed quests for instance, which mixes several artists in one quest.
+
+If you're curious and you want to know more about the Guild, please visit [the official Mappers' Guild website](https://mappersguild.com)!
+
+From now on, on a monthly basis, we'll be posting a newspost to keep you informed about the progress of the Mappers' Guild events, such as finished quests, but also some standalone sets that have been made by the Guild members.
+
+Considering there is a lot of finished quests, we will only list the ones that involves a lot of mapsets and mappers.
+
+![Cranky header image](https://assets.ppy.sh/artists/23/header.jpg)
+
+The **Cranky Hybrid Pack** quest got completed by **[Icekalt](https://osu.ppy.sh/users/5410645)**, **[Kalibe](https://osu.ppy.sh/users/3376777)**, **[Nao Tomori](https://osu.ppy.sh/users/5364763)**, **[Mao](https://osu.ppy.sh/users/2204515)**, **[Myxo](https://osu.ppy.sh/users/2202645)**, **[ScubDomino](https://osu.ppy.sh/users/8972308)**, **[xLolicore-](https://osu.ppy.sh/users/4525153)**, **[Nwolf](https://osu.ppy.sh/users/1910766)**, **[Ascendance](https://osu.ppy.sh/users/2931883)**, **[JBHyperion](https://osu.ppy.sh/users/4879508)**, **[Du5t](https://osu.ppy.sh/users/6053071)** and **[Jemzuu](https://osu.ppy.sh/users/7890134)**.
+
+- [Cranky - Conflict](https://osu.ppy.sh/beatmapsets/936126) by [Icekalt](https://osu.ppy.sh/users/5410645)
+- [Cranky - Amanojaku](https://osu.ppy.sh/beatmapsets/978694) by [Mao](https://osu.ppy.sh/users/2204515)
+- [J. Pachelbel - Canon (Cranky Remix)](https://osu.ppy.sh/beatmapsets/978674) by [Nao Tomori](https://osu.ppy.sh/users/5364763)
+- [L. V. Beethoven - Moonlight Sonata (Cranky Remix)](https://osu.ppy.sh/beatmapsets/978759) by [Nwolf](https://osu.ppy.sh/users/1910766)
+- [Cranky - EMPEROR](https://osu.ppy.sh/beatmapsets/979438) by [JBHyperion](https://osu.ppy.sh/users/4879508)
+- [Cranky - JULIAN](https://osu.ppy.sh/beatmapsets/1008310) by [ScubDomino](https://osu.ppy.sh/users/8972308)
+
+![The Flashbulb header image](https://assets.ppy.sh/artists/37/header.jpg)
+
+The **Flashbulb Standard Beatmap Pack** quest got completed by **[Pachiru](https://osu.ppy.sh/users/2850983)**, **[Gabe](https://osu.ppy.sh/users/654108)**, **[Zetera](https://osu.ppy.sh/users/587737)**, **[polka](https://osu.ppy.sh/users/4191702)**, **[greenhue](https://osu.ppy.sh/users/4967662)**, **[Irreversible](https://osu.ppy.sh/users/1287964)**, **[MBomb](https://osu.ppy.sh/users/3071175)**, **[Mismagius](https://osu.ppy.sh/users/19048)**, **[Leader](https://osu.ppy.sh/users/631530)**, **[fieryrage](https://osu.ppy.sh/users/3533958)** and **[Deif](https://osu.ppy.sh/users/318565)**.
```

Comment:
Shouldn't it be **The Flashbulb** (w/ "the" bolded) instead of  The **Flashbulb** tho?

(i'm quite sure that The Flashbulb has never been known or referred to as only "Flashbulb" btw)